robottelo.properties.sample

```diff
@@ -104,6 +103,16 @@ ssh_key=
 # capsule_repo=http://capsule/repo
 # sattools_repo=http://sattools/repo
 
+# Bugzilla data bellow are used to skip tests which are not implemented
+[bugzilla]
+# buzilla URL
+# bz_url=https://bugzilla.redhat.com
```

Comment:
bugzilla url is going to be a constant - there is no need to have this in the robottelo.properties file.  I vote to leaving this in robottelo/decorators/__init__.py : BUGZILLA_URL = "https://bugzilla.redhat.com/xmlrpc.cgi"